triggers/service/BUILD.bazel

```diff
@@ -328,5 +328,3 @@ EOF
     tools = ["//compiler/damlc"],
     visibility = ["//visibility:public"],
 )
-
-exports_files(["release/trigger-service-logback.xml"])
```

Comment:
Turns out this is actually necessary.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/13295/commits/b1d44d66a942456dbbbb859a12a13cf7a4877f4b